Testing/SystemTests/tests/analysis/ISISIndirectBayesTest.py

```diff
@@ -294,10 +300,8 @@ def runTest(self):
         background = 'Sloping'
         loopOp = False
 
-        spath = sname+'.nxs'    # path name for sample nxs file
-        LoadNexusProcessed(Filename=spath, OutputWorkspace=sname)
-        rpath = rname+'.nxs'    # path name for res nxs file
-        LoadNexusProcessed(Filename=rpath, OutputWorkspace=rname)
+        LoadNexusProcessed(Filename=sname, OutputWorkspace=sname)
```

Comment:
Is this correct? The old version loaded `spath` which was sname + `'.nxs`...

Reply:
I've tweaked it back to the filename plus ext to be safe :+1: